### PR TITLE
add basic support for dumping solidity files out during build

### PR DESCRIPTION
**Pull request by vessenes**
I'd love a code review and comments. This was built as part of the MakerDAO audit, and it's super handy to be able to get the solidity files out of the build process.

### CODE REVIEW

**Comment by vessenes** [comment]
didn't run npm test, my bad.

File: cmd/main.js [hunk 1]
```diff
@@ -179,7 +179,9 @@ if (cli.install) {
       name: workspace.dappfile.name,
       nameFilter: nameFilter,
       include_tests: cli['--tests'],
-      subpackages: cli['--subpackages'] || cli['-s']
+      subpackages: cli['--subpackages'] || cli['-s'],
+      dumpFile: cli['--dumpFile'],
+      dumpDir: cli['--dumpDir']
```
Comment (dbrock) [inline cmd/main.js]: So sorry for trivial comment but these should be `--dump-file` and `--dump-dir`

**Comment by dbrock** [comment]
I like this! :+1: I've been wishing this existed a few times already actually, but not enough to actually go ahead and add it!

**Comment by dbrock** [comment]
Also btw nobody likes the semistandard bullshit, if you make another pull request to remove that from the build you would be considered a hero among all the other developers

**Comment by vessenes** [comment]
I have to leave the reducing standardization pull requests to someone else :)

Let me know if these arg names are what you want.

**Comment by dbrock** [comment]
Haha great thanks :+1:

**Comment by ryepdx** [comment]
Regarding semistandard: it was an early choice made by a consensus of the two or three people working on it at the time, and certainly can and should be changed if it's no longer serving those working on dapple. I think it's useful for projectsto have a consistent coding style, but I don't have any particular opinion on what that should be.

**Pull request merged**